ToolBox/Polygon/PolygonSIPMappings.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuantConnect.ToolBox.Polygon
+{
+    // Provides information for SIP sale conditions according to the Polygon Data Vendor
+    // https://polygon.io/glossary/us/stocks/conditions-indicators
+    //
+    public static class PolygonSIPTradeMappings
```

Comment:
We should modify this class so it maps Polygon trade conditions into Leans format see https://github.com/QuantConnect/Lean/pull/5227/files and those values should be the ones to be stored in the `Tick.SaleCondition`